mlflow/azureml/__init__.py

```diff
@@ -206,8 +207,163 @@ def build_image(model_uri, workspace, image_name=None, model_name=None,
             image.wait_for_creation(show_output=True)
         return image, registered_model
 
+@experimental
+def deploy(model_uri, workspace, deployment_config = None, service_name=None, model_name=None,
+                mlflow_home=None, synchronous=True):
+    """
+    Register an MLflow model with Azure ML and build an Azure ML ContainerImage for deployment.
```

Comment:
@csteegz This documentation needs to be updated; it is currently identical to `build_image`.